src/views/Article.vue

```diff
@@ -56,7 +57,6 @@
       onSelectionChange: function () {
         const s = document.getSelection()
         if ( s.isCollapsed ) {
```

Comment:
I don't know if I missed this when testing #23 but setting the text to empty string here was causing it not render later over the image

Reply:
Could you elaborate or maybe report the issue on phab if it's a real bug?

Reply:
On it https://phabricator.wikimedia.org/T295047